WordPress/Classes/Models/BlockEditorSettings+GutenbergEditorSettings.swift

```diff
@@ -81,7 +85,15 @@ extension BlockEditorSettings {
                                                          type: .experimentalFeatures,
                                                          order: 0,
                                                          context: context)
+        let quoteKey = BlockEditorExperimentalFeatureKeys.quoteBlockV2.rawValue
+        let quoteRefactor = BlockEditorSettingElement(name: quoteKey,
+                                                         value: "\(remoteSettings.quoteBlockV2)",
```

Comment:
I'm sure this is just me not knowing Swift 🙂  .. why do we need to escape the opening `(`, but not the closing `)`?

Reply:
It's called [string interpolation](https://docs.swift.org/swift-book/LanguageGuide/StringsAndCharacters.html#ID292) I had to investigate this as well while I was using another PR as a reference to create this one 😅

Reply:
Ah, it's not escaping, I get it. Thanks for explaining! 🙇